pkg/sdk/testint/tables_integration_test.go

```diff
@@ -1014,6 +1014,24 @@ func TestInt_TablesShowByID(t *testing.T) {
 		t.Cleanup(cleanupTableHandle(id))
 	}
 
+	uploadFile := func(t *testing.T, stage *sdk.Stage) string {
```

Comment:
we have a function putOnStage (we had it before in helpers_test.go but now it was moved as part of the helpers refactor and can be accessed by `testClientHelper().Stage.PutOnStage(t, stage.ID(), "<path to file in testdata dir>")`

Reply:
Done.